geonetwork/gn3-migration/add-logo-categories/README.md

```diff
@@ -0,0 +1,116 @@
+           <mapping sourceId="1d54120f-ab3f-409f-959a-03580d2827f2" category="portal:AODN"></mapping>
+           <mapping sourceId="08be49a2-78e8-401b-8c8f-6f1417dd9185" category="portal:AODN"></mapping>
+           <mapping sourceId="7ee2e527-efae-405a-b36e-a49183d60ece" category="portal:DSTO"></mapping>
+           <mapping sourceId="96425e8d-d1e1-4bd0-92d9-b9b2f3d2fbeb" category="portal:AAD"></mapping>
+           <mapping sourceId="052d79e4-be0b-4124-81a4-724a5862c40a" category="portal:AAD"></mapping>
+           <mapping sourceId="794c8c8e-fa0b-47c8-b962-3d992de2fdd4" category="portal:AIMS"></mapping>
+           <mapping sourceId="b230d39a-c1cb-4efa-bbad-174b8d74fba0" category="portal:IMAS"></mapping>
+           <mapping sourceId="11caadf5-d52e-46c1-b227-e6bfbca0b919" category="portal:OEH"></mapping>
+           <mapping sourceId="99ea3014-8de8-4cae-ad39-b59b5dcc93c5" category="portal:UWA"></mapping>
+           <mapping sourceId="a4cc61f5-ec10-41b4-90ae-e72bb9ca278b" category="portal:ACE"></mapping>
+           <mapping sourceId="e0f13c0c-5473-4959-8c08-6767edf8ce2c" category="portal:AODN"></mapping>
+           <mapping sourceId="3961fb14-12fb-4575-9ee2-386e66f8bfc6" category="portal:NIWA"></mapping>
+           <mapping sourceId="9707b912-1ddd-463f-a57f-487482b71e94" category="portal:AIMS"></mapping>
+           <mapping sourceId="a79ab1a1-0afe-4cae-b6f8-e8a058d74508" category="portal:RLS"></mapping>
+           <mapping sourceId="ba266bbb-5dba-40a5-9a96-deadd823c5ca" category="portal:GA"></mapping>
+           <mapping sourceId="b4f8e3b8-f527-42fd-9966-030139513c64" category="portal:GA"></mapping>
+           <mapping sourceId="4c677113-1875-4a3d-b7e8-84d6d2f019ce" category="portal:IMAS"></mapping>
+           <mapping sourceId="69739489-42f5-4ed0-af75-c6c0aefa8db0" category="portal:IMAS"></mapping>
+           <mapping sourceId="bf5999d9-8c47-408e-a25b-19fc898ab39d" category="portal:IMAS"></mapping>
+           <mapping sourceId="997c2ceb-a58a-4df6-8dd5-2fbb09111809" category="portal:IMAS"></mapping>
+           <mapping sourceId="ed23e365-c459-4aa4-bbc1-5d2cd0274af0" category="portal:IMOS"></mapping>
+           <mapping sourceId="b6718c9f-f6e6-467e-8721-01efbc00b7da" category="portal:OEH"></mapping>
+           <mapping sourceId="8fc19f74-e4bc-4401-9103-00b3624aefba" category="portal:OEH"></mapping>
+           <mapping sourceId="cb4f24e1-4e60-420d-8eac-ad57f9c070fd" category="portal:NOAA"></mapping>
+           <mapping sourceId="fa56df7a-6a9c-4ead-a191-36fdf8faba5c" category="portal:RAN"></mapping>
+       </mappings>
+   </xsl:variable>
+    
+    <xsl:template match="/">
+        <xsl:for-each select="response/metadata">
+            <xsl:variable name="source_uuid" select="source"/>
+            <xsl:value-of select="concat(*/uuid, ',', source, string($mapping/mappings/mapping[@sourceId=$source_uuid]/@category), '&#x0A;')"/>
```

Comment:
As discussed, there appears to be an extra `source` here. It should be changed to
```bash 
<xsl:value-of select="concat(*/uuid, ',', string($mapping/mappings/mapping[@sourceId=$source_uuid]/@category), '&#x0A;')"/>```

Reply:
Process to create mapping between uuid and category is added and Readme file revised accordingly.